### ISSUE TITLE
What is the difference between valid_nll_loss and nll_loss?

### ISSUE THREAD

**Issue by asispatra** [pytorch/fairseq#559]
Earlier I use to ` valid_nll_loss` in the validation:
```
| epoch 001 | valid on 'valid' subset | valid_loss 10.0274 | valid_nll_loss 9.16604 | valid_ppl 574.45 | num_updates 378
```
But now it is `nll_loss` in the validation:
```
| epoch 001 | valid on 'valid' subset | loss 10.073 | nll_loss 9.216 | ppl 594.56 | num_updates 368
```
What is the difference between valid_nll_loss and nll_loss in the log?

**Comment by myleott** [comment]
It was a recent change in the logging format to support tensorboard. It's the same as it was before (i.e., the nll_loss on the validation set).
